src/components/Error/Error.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+const Error = () => {
+  return (
+    <>
+      <h4>Thank you for visiting our website.</h4>
+      <p>You've found a page that we cannot find.</p>
+
+      <p>Please return to our 🏡 home page using the Tree Pal link above.</p>
```

Comment:
It is a good touch to pick one of the house emojis with the tree for our project,tree-pal!